core/domain/classifier_domain.py

```diff
@@ -277,6 +277,27 @@ def status(self):
     def training_data(self):
         return self._training_data
 
+    def update_state_name(self, state_name):
```

Comment:
@pranavsid98 oh I'm really sorry, I commented on the right method but used the wrong word. It is the state_name that I'm talking about and not the status. Sorry, I corrected it.